### ISSUE TITLE
Change from "Stick to the to of the blog" to "Pin to the top of the blog"

### ISSUE THREAD

**Issue by deborah86** [WordPress/gutenberg#41316]
## What problem does this address?

The language is inconsistent with the language used in many other popular apps. Creating your own language might confuse new users. Most users are more familiar with the term "Pin" than "Stick".
## What is your proposed solution?

The option should be changed from "Stick to the to of the blog" to "Pin to the top of the blog"

![image](https://user-images.githubusercontent.com/12677420/170261582-ae118bde-332c-400d-8cfd-c7a99dbe1f9d.png)

**Comment by kristastevens** [comment]
Hello from Editorial! 👋 

Where does this appear in the UI? Is this the only place that this phrase appears in the UI? 

No concerns with the change in language, though we should be sure we're being consistent throughout.

**Comment by deborah86** [comment]
@kristastevens It appears in the post editor. The text is not clear for new users. One of the main complaints about WordPress is that it is not user-friendly. The options are hard to understand. Pin would be a better term to use over stick because various websites use pin.

![image](https://user-images.githubusercontent.com/12677420/170278296-87743a7b-6e48-43f3-8d02-e2c2cc2cda80.png)

**Comment by kristastevens** [comment]
While at first the change seems innocuous, it goes deeper than adjusting the language in the UI. This nomenclature is based on the "[Sticky Posts](https://wordpress.com/support/post-and-page-visibility/#sticky-posts-posts-only)" feature. That would need to be renamed and the support doc / video updated to reflect that, though revising resources shouldn't be a barrier to improvement.

What data do we have that suggests that "stick" is unclear to users? I know that Pinterest, Facebook, and Twitter use "pin" though "pin" and "stick" are synonyms when used as a verb. The person who makes the determination on whether or not this should change would likely want to understand the data points that point to user confusion over this before committing to it.

**Comment by deborah86** [comment]
@kristastevens The pin terminology is more common. Even though both are synonyms, it is always better to go with the terminology that is more common.

Can you give examples of platforms as popular as YouTube, Facebook, Twitter, Reddit, Medium, etc. that use the "stick" terminology besides WordPress?

**Comment by kristastevens** [comment]
@deborah86 I agree that the "pin" terminology is more common. That in and of itself doesn't make the word "stick" unclear in this context. Without any data from users that suggests that "stick" as a term is confusing, I would recommend leaving it as is.

**Comment by deborah86** [comment]
If you are trying to target new users who are unfamiliar with WordPress it is better to use more common terminology instead of trying to reinvent the wheel. In conversations, I haven't heard anyone use the term stick. It is usually the term pin. The point of changing this term is to make it easier for new users to transition from other platforms to WordPress. Using common terminology that is found on most other platforms will help with this transition.

**Comment by Mamaduka** [comment]
Closing in favor of #69300, which has more recent discussions around the subject.
